features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/admin_new/source/src/app/components/AdminPages/Addons/ListBase.jsx

```diff
@@ -64,7 +64,7 @@ const useStyles = makeStyles((theme) => ({
  */
 function ListLabels(props) {
     const {
-        EditComponent, editComponentProps, DeleteComponent, showActionColumn,
+        EditComponent, editComponentProps, DeleteComponent, showActionColumn, deleteComponentProps,
```

Comment:
Don't see the proType validation added for this prop. Might be failing the prod build.

Reply:
Fixed